apis/python/src/tiledbsoma/soma_indexed_dataframe.py

```diff
@@ -19,7 +19,7 @@ class SOMAIndexedDataFrame(TileDBArray):
     """
     Represents ``obs``, ``var``, and others.
 
-    A ``SOMAIndexedDataFrame`` contains a "pseudo-column" called ``soma_rowid``, of type uint64 and domain [0,num_rows).  The ``soma_rowid`` pseudo-column contains a unique value for each row in the ``SOMAIndexedDataFrame``, and is intended to act as a join key for other objects, such as a ``SOMASparseNdArray``.
+    All `SOMAIndexedDataFrame` must contain a column called `soma_joinid`, of type `uint64`. The `soma_joinid` column contains a unique value for each row in the `SOMAIndexedDataFrame`, and intended to act as a joint key for other objects, such as `SOMASparseNdArray`.
```

Comment:
let's use double-backtick -- this is important for python help-doc and readthedocs rendering -- sorry, it's a bit ugly here in the code but it'll look better downstream! :)

Reply:
fixed